### ISSUE TITLE
RuntimeError: Could not find 'xenon-grpc' jar file

### ISSUE THREAD

**Issue by arnikz** [MD-Studio/cerise#60]
I've got the following error while executing

`docker run --name=cerise -p 29593:29593 cerise`

```
 * Starting nginx nginx
   ...done.
Traceback (most recent call last):
  File "cerise/run_back_end.py", line 15, in <module>
    xenon.init()
  File "/usr/local/lib/python3.5/dist-packages/xenon/server.py", line 150, in init
    __server__.__enter__()
  File "/usr/local/lib/python3.5/dist-packages/xenon/server.py", line 80, in __enter__
    self.process = start_xenon_server(self.port, self.disable_tls)
  File "/usr/local/lib/python3.5/dist-packages/xenon/compat.py", line 39, in start_xenon_server
    raise RuntimeError("Could not find 'xenon-grpc' jar file.")
RuntimeError: Could not find 'xenon-grpc' jar file.
```

Should `xenon-grpc` be added to the list of dependencies?

**Comment by LourensVeen** [comment]
Oh, no, Xenon 2 got released, and there's probably no version check on pyxenon in the requirements.txt, as a result of which it's trying to build with Xenon 2, while Cerise hasn't been ported yet. So the short-term fix is to fix the requirements to require Xenon 1, while the longer-term fix (starting next week) is to port Cerise to Xenon 2.

**Comment by LourensVeen** [comment]
Fixed by 2f1f52b
